include/aspect/global.h

```diff
@@ -39,6 +39,7 @@
 
 #include <deal.II/base/mpi.h>
 #include <deal.II/base/multithread_info.h>
+#include <deal.II/base/std_cxx11/unique_ptr.h>
```

Comment:
Do you think it is necessary to include the unique_ptr here? It is only used in a few places, so should we include it only there? I am just asking, because it somehow feels out of place here (there is no inclusion of the other cxx11 features).